swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory == None else inventory 
+    
+    def add(self, item):
+        self.inventory.append(item)
+        
+        return item
+    
+    def remove(self, item):
+        result = False
+
+        if item in self.inventory:
+            self.inventory.remove(item)
+            result = item
+
+        return result
+    
+    def get_by_id(self, item_id):
+        results = None
+
+        for item in self.inventory:
+            if item.id == item_id:
+                results = item
+                break
+
+        return results
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        
+        result = False
+
+        if my_item in self.inventory and their_item in other_vendor.inventory:
+                self.remove(my_item)
+                other_vendor.remove(their_item)
+                self.add(their_item)
+                other_vendor.add(my_item)
+                result = True
+        return result
```

Comment:
I would add a space between these two lines of code so you can better visualize the difference in responsibility between the two (This is something you will do in general). One section's responsibility is adding and removing items from vendors' inventories and the other section is returning the result of the function.
```suggestion
                result = True
                
        return result
```